src/Extensions/NeoServer.Scripts.LuaJIT/Functions/ActionFunctions.cs

```diff
@@ -23,6 +23,49 @@ public void Init(LuaState luaState)
         RegisterMethod(luaState, "Action", "aid", LuaActionItemActionId);
         RegisterMethod(luaState, "Action", "uid", LuaActionItemUniqueId);
         RegisterMethod(luaState, "Action", "allowFarUse", LuaActionAllowFarUse);
+        RegisterMethod(luaState, "Action", "blockWalls", LuaActionBlockWalls);
+        RegisterMethod(luaState, "Action", "checkFloor", LuaActionCheckFloor);
+    }
+    
+    /// <summary>
+    /// action:checkFloor(bool)
+    /// ref: int LuaScriptInterface::luaActionCheckFloor(lua_State* L) luascript.cpp
+    /// </summary>
+    /// <param name="l"></param>
+    /// <returns></returns>
+    public static int LuaActionCheckFloor(LuaState l)
+    {
+        var action = GetUserdata<Action>(l, 1);
+        if (action is not null) {
+            action.SetCheckFloor(GetBoolean(l, 2));
+            PushBoolean(l, true);
+        } else {
```

Comment:
Can we avoid using "else" when possible?

Reply:
yeah of course, this scenario doesn`t was used because always return  1